config/printer-ratrig-v-minion-2021.cfg

```diff
@@ -0,0 +1,178 @@
+# This file contains pin mappings for the Ratrig V-Minion
+# with an Octopus Pro v1.1 board.
```

Comment:
Please state that this is for the "full kit" version, as configs should only describe stock hardware that an end user would receive, we do not accept DIY printers due to the massive amount of variation that can be involved. If this config describes one of those DIY variants then we would be inclined not to accept this addition.

You may also want to state that this config will not have the full out of the box functionality of the stock ratos version of klipper.

Thanks
James